app/cypress/integration/reporter-facilities-list.spec.js

```diff
@@ -15,11 +15,7 @@ describe('When the reporter is viewing a list of facilities & applications', ()
       'have.value',
       2019
     );
-    cy.get('tbody > tr:nth-child(1) .btn').should(
-      'have.prop',
-      'disabled',
-      false
-    );
```

Comment:
@matthieu-foucault It appears that the Resume button is never disabled anywhere anyway, so it's a bit funny that we were using this assertion.

Anchor links don't have a `disabled` prop, which is why it was failing. Since the intention of this test was to ensure that the button/link is active, I've replace with an assertion that it has an `href` although I'm not even sure that's necessary

Reply:
That assertion might have been relevant at some point in the past